Standards/CCSMB-11.md

```diff
@@ -0,0 +1,99 @@
+| CompliantWindowManagement | bool                        | Whether the application manages its own windows through a system-specific API. If set to `false`, compliant specifications SHOULD define `ShowIn` (see below). If `Terminal` is set to `true`, this is implied to be `true`. |
+
+Compliant application specifications MAY define the following additional fields:
+
+| Field       | Type                        | Description                                                                                                                                                                                                             |
+|-------------|-----------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Icon        | str-indexed array of str    | An array, where the keys are sizes in pixels (16x16 and 32x32 are common) and the values are paths to a square image representing the application. The file MAY be a BIMG file; other formats are permitted but discouraged.                                                                                      |
+| GenericName | str                         | The "generic name" of the application. For example, for a file browser, this should be "File Browser".                                                                                                                                                                                                            |
+| Categories  | number-indexed array of str | Categories in which the application should be shown in application launchers. For a list of valid values, see the [FreeDesktop Desktop Menu specification](https://www.freedesktop.org/wiki/Specifications/menu-spec/).                                                                                           |
+| Keywords    | number-indexed array of str | Keywords which, when combined with Name and GenericName, are useful for searching valid applications.                                                                                                                                                                                                             |
+| ShowIn      | number-indexed array of str | A list of target software that the application specification should be listed in. Compliant target software SHOULD ignore the specification if this field is set and the target software is not listed here.                                                                                                      |
+
+Compliant software MAY define extra fields, but they MUST prefixed with `X-`. Compliant software SHOULD ignore all unrecognized fields without the prefix.
```

Comment:
starting a key with `X-` would require doing `["X-something"] = "something"`, use `X_` so you can do `X_something = "something"`

Reply:
> starting a key with `X-` would require doing `["X-something"] = "something"`, use `X_` so you can do `X_something = "something"`

I've turned it into an array named `Extra`.